fe/fe-core/src/main/java/com/starrocks/common/proc/ProcService.java

```diff
@@ -71,6 +71,7 @@ private ProcService() {
         root.register("catalog", GlobalStateMgr.getCurrentState().getCatalogMgr().getProcNode());
         root.register("compactions", new CompactionsProcNode());
         root.register("warehouses", new WarehouseProcDir(GlobalStateMgr.getCurrentWarehouseMgr()));
+        root.register("compaction_num", new CompactionProcDir(GlobalStateMgr.getCurrentSystemInfo()));
```

Comment:
There is already a `root.register("compactions", new CompactionsProcNode());`
It should be merged with the two labels.

Reply:
But their forms have different columns, so it’s difficult to merge them into one.